.github/workflows/broken-link-check.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  push:
+  schedule:
+    - cron: 0 0 * * * # run daily at midnight
```

Comment:
I get what happens when the check fails on a push, but what happens when the scheduled check fails?  Is there a way to make it send a notification or push to Splunk or something along those lines?

Otherwise, seems good to me.